internal/otelcollector/config/metricagent/config_builder_test.go

```diff
@@ -22,9 +22,11 @@ func TestBuildConfig(t *testing.T) {
 	tests := []struct {
 		name                string
 		goldenFileName      string
-		pipelines           []telemetryv1alpha1.MetricPipeline
-		istioEnabled        bool
 		overwriteGoldenFile bool
+		pipelines           []telemetryv1alpha1.MetricPipeline
+
+		// istioActive indicates if the "cluster" has an active istio installation or not. Not to be confused with the IstioInput in a pipeline
```

Comment:
Can you maybe move this comment to the field documentation in the BuildOptions?

Reply:
I added a similar one